docs/03-tutorials/00-api-exposure/apix-03-set-up-tls-gateway.md

```diff
@@ -70,9 +70,7 @@ To create a TLS Gateway in simple mode, run:
             cipherSuites:
```

Comment:
cipherSuites and minProtocolVersion can be removed completely as we rely on Istio / Envoy defaults